db/skipped_sequence_test.go

```diff
@@ -133,35 +228,146 @@ func TestRemoveSeqFromSkipped(t *testing.T) {
 			remove:       100,
 			errorRemove:  150,
 		},
+		{
+			name:                 "list_full_range_items",
+			inputList:            []uint64{5, 15, 25, 35, 45},
+			expectedStartSeqList: []uint64{5, 15, 25, 28, 35, 45},
+			expectedEndSeqList:   []uint64{10, 20, 26, 30, 40, 50},
```

Comment:
What do you think about making these test definitions a bit more readable, so that it's more immediately obvious what the input ranges are and what the expected results are?  In particular the fact that we're creating ranges of [input, input+5] isn't really obvious, and it's also hard to eyeball what the expected end ranges are.

I was thinking of something like:
```
input: [][]uint64{{5,10}, {15,20}, {25,30}, {35,40}, {45,50}}
expected: [][]uint64{{5,10}, {15,20}, {25,26}, {28,30}, {35,40}, {45,50}
```
With that approach you could use {25} for cases where we expect a single sequence, and {25,25} where we expect a range of length 1.  You could also include the range len=2 and range len=3 cases in this test a bit more easily.  

It might be personal preference, if you disagree you can ignore.

Reply:
However, want to ensure we've got coverage for all permutations of removals, including:
- length 1 range
- length 2 ranges
- length 3 ranges, for removal of first/second/third (I think test below just removes the middle element)

Reply:
I agree, have made changes to have input and expected be 2-D array of uint64[][]. 

Added test cases to the test `TestRemoveSeqFromSkipped` for length 1 range seq removal, and length 2 seq removal. Have extended test `TestRemoveSeqFromThreeSequenceRange` to include a test for removing the start/last seq on a three sequence range.